data/yaml/joint.yaml

```diff
@@ -640,6 +640,8 @@ louhi:
 - W16-61
 - W18-56
 - D19-62
+lrec:
+  2020: [2020.aespen-1, 2020.ai4hi-1, 2020.bucc-1, 2020.calcs-1, 2020.cllrd-1, 2020.clssts-1, 2020.cmlc-1, 2020.computerm-1, 2020.framenet-1, 2020.gamnlp-1, 2020.globalex-1, 2020.isa-1, 2020.iwltp-1, 2020.ldl-1, 2020.lincr-1, 2020.lr4sshoc-1, 2020.lt4gov-1, 2020.lt4hala-1, 2020.mmwn-1, 2020.multilingualbio-1, 2020.onion-1, 2020.osact-1, 2020.parlaclarin-1, 2020.rail-1, 2020.readi-1, 2020.restup-1, 2020.sltuccurl-1, 2020.stoc-1, 2020.trac-1, 2020.wac-1, 2020.wildre-1]
```

Comment:
To get them listed under the expected place on the website, these should also be joint with `ws`.

The way we mark something as a workshop _really_ needs a revamp now with the new-style IDs...

Reply:
Do we need to add them there anymore? It's always been a bit confusing because "WS" is in the ACL section, but these are non-ACL. Since they're joint with LREC, they're findable from there.

Reply:
But that has always been true for all workshops. If we don't add all workshops to WS, I don't know what its purpose would be, since if someone thinks "there was a workshop in 2020 about this ..." and they click WS20 on the front page, they won't find it.

Reply:
I mean, I agree with the issues with this, but I believe workshops should still be marked somehow, and making them joint with "WS" is currently the only means we have for this.

Looking forward, maybe we could introduce an `is_workshop` flag somewhere and use that e.g. when refactoring the front page.